cmd/thanos/flags.go

```diff
@@ -12,13 +12,17 @@ import (
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
-func regCommonServerFlags(cmd *kingpin.CmdClause) (*string, *string, func(log.Logger, *prometheus.Registry, bool, string, bool) (*cluster.Peer, error)) {
+func regCommonServerFlags(cmd *kingpin.CmdClause) (*string, *string, *bool, *string, *string, func(log.Logger, *prometheus.Registry, bool, string, bool) (*cluster.Peer, error)) {
```

Comment:
The amount of return args here is massive. Can we name them and use in vars below? (Also you would format them in newlines)

Reply:
sounds sensible. I tried reducing the number, but the regCommand/runCommand seperation made it tricky. Obviously this'll get worse if we want service specific certs/keys.

Reply:
Added names for the return params, and split accross lines, let me know if you meant something else.